### DISCUSSION TITLE
How do you handle multiple developer accounts with 2FA?

### DISCUSSION THREAD

**Discussion by dinsen** [fastlane/fastlane#18152] [Q&A]
Before February it was no problem to use the same phone number on multiple Apple ID's. But now that Apple wants us to enable 2FA on Apple ID's our phone number can't received 2FA codes. 
I tried to enable 2FA on an account and got an error that the phone number was used to many times. 
(We have around 10 Apple ID's with the same phone number)

We manage 150+ developer accounts and I really don't want to buy a phone number for each of them. 

There is also the possibility to only have 1 Apple ID added to all 150+ developer accounts. But it really screams for security issues doing that.

How do you handle 2FA with multiply developer accounts?

**Comment by airtonix** [comment]
My feeling is that you should be :
- using the new api_key method.
- have a service account on playstore and applestore using a keyfile and apiKey repsectively.
- use codesigning: cryptex for android and match/sigh/cert for ios 
- have different codesigning repos for dev and prod
- use something like https://github.com/atlassian/changesets for your release process (and automation around that)

with this in play, you only need two accounts per store. your developer specific accounts should only be used for local dev.
